build.sbt

```diff
@@ -45,9 +45,19 @@ val noPublish = List(
 )
 
 val mimaSettings = Def settings (
-  mimaPreviousArtifacts := Set(organization.value %% moduleName.value % "0.8.0-M4")
+  mimaPreviousArtifacts := Set(organization.value %% moduleName.value % "0.8.0-M4"),
+  mimaBinaryIssueFilters ++= ignoredABIProblems
 )
 
+val ignoredABIProblems = {
+  import com.typesafe.tools.mima.core._
+  import com.typesafe.tools.mima.core.ProblemFilters._
+  Seq(
+    exclude[ReversedMissingMethodProblem]("sjsonnew.CalendarFormats.offsetDateTimeStringIso"),
+    exclude[ReversedMissingMethodProblem]("sjsonnew.CalendarFormats.sjsonnew$CalendarFormats$_setter_$offsetDateTimeStringIso_=")
```

Comment:
How do you feel about establishing a practice where exclusions (like these) must include a comment justifying them? I think it's a good practice to have, if not just to avoid miss-understandings over the reasoning.